### PR TITLE
add the cloud-resource-operator and bump the integreatly version

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation
Add the cloud-resource-operator to the integreatly operator, needed for HA, 

## What
add the cloud-resource-operator and bump the integreatly version 

## Verification Steps
1. Push the changes above to your app registry
 - Run `make push/all`
 - **Verify**: cloud-resoure (0.1.0) and Integreatly Operator (1.7.6) should be available in your app registry
2. Create an `OperatorSource` with your app registry
e.g.
```yaml
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  finalizers:
    - finalizer.operatorsources.operators.coreos.com
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators austincunningham
  endpoint: 'https://quay.io/cnr'
  publisher: austincunningham Publisher
  registryNamespace: austincunningham
  type: appregistry
```
 - **Verify**: CRO operator should be deployed
## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

### CODE REVIEW

**Comment by dimitraz** [comment]
:eyes:

File: integreatly-cloud-resources/cloud-resources-0.1.0/cloud-resource-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,239 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (dimitraz) [inline integreatly-cloud-resources/cloud-resources-0.1.0/cloud-resource-operator.clusterserviceversion.yaml]: I think the filename needs to include the cloud resources version as well

File: integreatly/integreatly-1.7.6/installation.crd.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (dimitraz) [inline integreatly/integreatly-1.7.6/installation.crd.yaml]: Does this need to be committed? I don't see an `integreatly` manifest on the master branch, so I'm assuming it's kept in the operator repo

File: integreatly/integreatly-1.7.6/integreatly-operator.1.7.6.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,366 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: integreatlyoperator.1.7.6
```
Comment (dimitraz) [inline integreatly/integreatly-1.7.6/integreatly-operator.1.7.6.clusterserviceversion.yaml]: I think the version in the integreatly-operator would also require a bump, looks like the version there is currently `1.12.0`

**Review by dimitraz — COMMENTED** [review]
A few comments

File: integreatly/integreatly-1.7.6/installation.crd.yaml (hunk 2 above)
Comment (dimitraz) [reply integreatly/integreatly-1.7.6/installation.crd.yaml]: Just checked with Jameel, she confirmed

Comment (austincunningham) [reply integreatly/integreatly-1.7.6/installation.crd.yaml]: A lot of changes in 3 day :+1:

File: integreatly/integreatly-1.7.6/integreatly-operator.1.7.6.clusterserviceversion.yaml (hunk 3 above)
Comment (austincunningham) [reply integreatly/integreatly-1.7.6/integreatly-operator.1.7.6.clusterserviceversion.yaml]: as not including it doesn't matter

**Review by dimitraz — APPROVED** [review]
Looks good

**Pull request merged**